### DISCUSSION TITLE
Oauth - How to?

### DISCUSSION THREAD

**Discussion by TarcisioMenezes** [dtcooper/raspotify#683] [Q&A]
Guys help me out here please. I'm kinda lost. It seems that Spotify doesn't support username and password auth anymore.
ct 23 13:40:47 pi librespot[3741]: [2024-10-23T18:40:47Z ERROR librespot] Invalid `--password` / `-p`: Password authentication no longer supported, use OAuth

My question is: Is Oauth Implemented on Raspotify 0.45? Can't find how I can authenticate my spotify account with raspotify.
Thank you!

**Comment by kimtore** [comment]
Hi @TarcisioMenezes, please check out the comment in the other oauth issue: https://github.com/dtcooper/raspotify/issues/675#issuecomment-2375392925.

**Reply by TarcisioMenezes** [reply]
hey @kimtore ! So it's not still implemented officially in raspotify, right? i.e, we could just add the spotify token for Oauth in the config file and it would work. is there any plan to add it? 
Thank you for you reply!
